Traceback (most recent call last):
  File "E:\Programming\Python\BattleKiller2D\main.py", line 909, in __init__
    self.s.connect((sip, sport))
ConnectionRefusedError: [WinError 10061] Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение
